.gitignore

```diff
@@ -15,3 +15,4 @@ pom.xml.asc
 *.csv
 *.xlsx
 *.xlsx
+*.tsv
```

Comment:
better to ignore a results directory so we can have some dummy data for testing?